boa_engine/src/syntax/parser/expression/left_hand_side/mod.rs

```diff
@@ -70,32 +72,50 @@ where
     type Output = Expression;
 
     fn parse(self, cursor: &mut Cursor<R>, interner: &mut Interner) -> ParseResult<Self::Output> {
+        fn is_super_call<R: Read>(
```

Comment:
Is it possible to add some documentation to explain why is this needed and how it detects the super call?

Reply:
Added docs